### ISSUE TITLE
Unable to configure options for any tracker - Buttons not working

### ISSUE THREAD

**Issue by 4C3T** [Jackett/Jackett#11794]
### Do not delete the template.

**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed. Also check our [Troubleshooting](https://github.com/Jackett/Jackett/wiki/Troubleshooting) for steps to resolve common issues.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**:
Linux Mint 20.1 Ulyssa 64-bit Mate 1.24.0
**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**:

**Jackett Version**:
v0.18.145
**Last Working Jackett Version**:

**Are you using a proxy or VPN?** [yes/no]
Yes
### Description
Unable to click the configure-buttons for each (and everyone) of the indexers, whilst on another machine it works just fine.
I cannot find any differences on chmod or chown between the two machines, although, they run on different account names.
At first when I installed (or more like transplanted from the fully working machine) it ran just fine with no problems, but about perhaps a couple of weeks ago it stopped working, and I figured it was Firefox that had a glitch and didn't think much of it as it does what it's suppose to do, to find results when searching.

Today I had a small crash and I went to check the logs, and found I had a permissions error when auto-update ran, so I fixed that, I think. Not seen any more errors about that, so  I took a stab at trying to compare the file/folders permissions between the two machines, and it does seem to match to the corresponding accounts, but I'm still unable to click those configure buttons for each indexer, which normally pops up a window with info about download options and valid search categories.

I'm going to replace that server in the near future and re-install everything, but it would be nice to know if anyone has any thoughts about what could cause this problem?

It does seem, at first glance that it's some file permission that's wrong, and they're set at 644 which should work I think.
Some are even 755, and all files, including the config-folder has the corresponding user/group, and the systemd uses the same credentials.
Another thought was that Firefox had screwed up something, but when I try accessing the other machine also running Jackett it works just fine. I also tried using another browser (also Firefox, on the actual machine) and when clicking the config-button, nothing happens.
Most peculiar, and I can't find anything that would indicate something wrong in the logs.
I'm not a Linux expert by any means, FYI. Been using it on and off for the past 20 years, never any serious use during those years, just simple single-purpose servers, starting getting serious with it about 6 months ago when I got fed up with Windows hogging all the resources, so I got tons of stuff still to learn.

Thanks in advance.

[List steps to reproduce the error and details on what happens and what you expected to happen]

### Logged Error Messages

[Place any relevant error messages you noticed from the logs here.]

[Make sure you attach the full logs with your personal information removed in case we need more information]
[log.txt](https://github.com/Jackett/Jackett/files/6534103/log.txt)
[updater.txt](https://github.com/Jackett/Jackett/files/6534116/updater.txt)

Note that the fix for permissions occurred sometime around 5 to 6 PM

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by ilike2burnthing** [comment]
Backup your config folder elsewhere, uninstall Jackett, and try a fresh reinstall (without restoring your backup).

If that works, stop Jackett, replace SeverConfig.json with the one from your backup, restart Jackett and see if things still work.

If so, stop Jackett again, replace the whole config folder, restart Jackett and see if things are working still.

**Comment by 4C3T** [comment]
Ok. Will try this at earliest convenience.
Thanks for the tip.

**Comment by 4C3T** [comment]
Ok, so I had another crack at it, and for some reason I seemed to have, should we say, "misplaced" the executable files permissions, which lead to this failure.
At first the buttons stopped working, as described above, but I could restart the service just fine, then a few days later that stopped working, and a look at the syslog revealed permission errors, so I went to the Jacket folder and checked the executables, and sure enough, their permissions had been changed so that my user account had no access to run it.

Now it works again.
I find file permissions very cumbersome to work with in Linux as there are so many ways things can go horribly wrong or not at all.
There are tons of different groups to choose from in some cases, just as there are in Windows, but only more and more confusing.
Thankfully, in most cases either trial and errors will solve it, or googling it does the trick. In some cases you just have to ask someone.

This time around, it sufficed with trial & errors.

So, thanks anyway.
